src/main/sensors/gyro.c

```diff
@@ -609,7 +668,7 @@ static uint16_t calculateNyquistAdjustedNotchHz(uint16_t notchHz, uint16_t notch
     return notchHz;
 }
 
-#if defined(USE_GYRO_SLEW_LIMITER)
+#ifdef USE_GYRO_SLEW_LIMITER
```

Comment:
What's the reason for this change? While not outright wrong, the long form has the advantage to convert into more complex expressions in a more seamless way.

Reply:
I was trying to make it consistent but I see it is mixed all over the code-base. I later learned that `if defined` can be chained in an expression. e.g.

```
#if defined(FOO) || defined(BAR)
```

whereas `ifdef` cannot.

I think I will switch back to `if defined().` form.

Reply:
in 90cd5da